cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -254,8 +254,8 @@ class SparkSQLExecuteImpl(val sql: String,
       case StringType =>
         TypeUtilities.getMetadata[String](Constant.CHAR_TYPE_BASE_PROP,
           f.metadata) match {
-          case Some(base) =>
-            val size = TypeUtilities.getMetadata[Long](
+          case Some(base) if base != "CLOB" =>
+            lazy val size = TypeUtilities.getMetadata[Long](
```

Comment:
This is related to "normalizeSchema" calls which transform these properties into something else (which was not clear to me why it was being done in the first place): https://github.com/SnappyDataInc/snappydata/blob/master/core/src/main/scala/org/apache/spark/sql/hive/SnappyStoreHiveCatalog.scala#L312

We are now avoiding normalization at the lowest level to allow quoted case-sensitive names to be passed through from SQL layer as is without changing case. The earlier code there would take out this property but no longer happens at that level. We could explicitly change the schema for incoming query (for saveAsTable) in SnappySession.createTableAsSelect which only normalizes names currently (see SnappySession.scala:1133) but it is not worth it since CLOB is no longer an issue at all with the thrift driver.